qa/sc_evm_mc_addr_ownership_multisig.py

```diff
@@ -0,0 +1,580 @@
+#!/usr/bin/env python3
+import json
+import pprint
+from decimal import Decimal
+
+from eth_abi import decode
+from eth_utils import add_0x_prefix, remove_0x_prefix, event_signature_to_log_topic, encode_hex, \
+    function_signature_to_4byte_selector, to_checksum_address
+
+from SidechainTestFramework.account.ac_chain_setup import AccountChainSetup
+from SidechainTestFramework.account.ac_utils import format_evm, estimate_gas
+from SidechainTestFramework.account.httpCalls.transaction.getKeysOwnership import getKeysOwnership
+from SidechainTestFramework.account.httpCalls.transaction.removeKeysOwnership import removeKeysOwnership
+from SidechainTestFramework.account.httpCalls.transaction.sendKeysOwnership import sendKeysOwnership
+from SidechainTestFramework.account.httpCalls.transaction.sendMultisigKeysOwnership import sendMultisigKeysOwnership
+from SidechainTestFramework.account.utils import MC_ADDR_OWNERSHIP_SMART_CONTRACT_ADDRESS, \
+    ZENDAO_FORK_EPOCH
+from SidechainTestFramework.scutil import generate_next_block, EVM_APP_SLOT_TIME
+from httpCalls.transaction.allTransactions import allTransactions
+from test_framework.util import (assert_equal, assert_true, fail, hex_str_to_bytes, assert_false,
+                                 forward_transfer_to_sidechain)
+
+"""
+Configuration: 
```

Comment:
Could you please add the tests for contract interoperability, as in `sc_evm_mc_addr_ownership.py`?

Reply:
Done